spec/lib/metasploit/framework/hashes/identify_spec.rb

```diff
@@ -18,320 +18,320 @@
 print("DES: %s") %(hash.des_crypt.hash("password"))
 =end
 
-RSpec.describe 'hashes/identify' do
+RSpec.describe Metasploit::Framework::Hashes do
   describe 'identify_md5' do
     it 'returns md5' do
-      hash = identify_hash('$1$IEHUWAxH$nMC1edxSFa4SaKH7hi2.P1')
+      hash = Metasploit::Framework::Hashes.identify_hash('$1$IEHUWAxH$nMC1edxSFa4SaKH7hi2.P1')
```

Comment:
Not a blocker, but, since the class is defined in `RSpec.describe` you can use `described_class` instead of the full class namespace:
```suggestion
      hash = described_class.identify_hash('$1$IEHUWAxH$nMC1edxSFa4SaKH7hi2.P1')
```